bentoml/_internal/bento/bento.py

```diff
@@ -334,6 +340,21 @@ def save(
             fs.mirror.mirror(self._fs, out_fs, copy_if_newer=False)
             self._fs.close()
             self.__fs = out_fs
+            event_properties = {
+                "bento_tag": str(self.info.tag),
+                "bento_creation_timestamp": self.info.creation_time.isoformat(),
+                "bento_size": human_readable_size(calc_dir_size(bento_path)),
+                "models_tags": [str(i) for i in self.info.models],
```

Comment:
model_tags, model_types?

Reply:
we want to track the model tag and types of the model in each bentos?

Reply:
we don't want to include the model tag